templates/init/hosting/index.html

```diff
@@ -10,13 +10,17 @@
     <!-- include only the Firebase features as you need -->
     <script defer src="/__/firebase/{{VERSION}}/firebase-auth.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-database.js"></script>
+    <script defer src="/__/firebase/{{VERSION}}/firebase-firestore.js"></script>
+    <script defer src="/__/firebase/{{VERSION}}/firebase-functions.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-messaging.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-storage.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-analytics.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-remote-config.js"></script>
     <script defer src="/__/firebase/{{VERSION}}/firebase-performance.js"></script>
     <!-- initialize the SDK after all desired features are loaded -->
     <script defer src="/__/firebase/init.js"></script>
+    <!-- connect to any running emulators -->
+    <script defer src="/__/firebase/initEmulators.js"></script>
```

Comment:
But this will cause an error in production when developers deploy it. Remembering to remove before deploy and add it back after deploy is really confusing.

Can we do something like `/__/firebase/init.js?useEmulatorsIfAvailable=true` instead and then remove that flag in the next CLI major release?

Reply:
Oh wow great point!  Ok I will do that.

Reply:
Changed and expanded PR description to include alternate proposal.